modules/CuckooFilter/src/main.rs

```diff
@@ -1,3 +1,57 @@
-fn main() {
-    println!("Hello, world!");
-}
+extern crate redis;
+use redis::Commands;
+use probabilistic_collections::cuckoo::CuckooFilter;
+
+fn main() {
+    let size = std::env::args().nth(1).expect("Please specify the size of the filter");
+    let action = std::env::args().nth(2).expect("Please specify an action");
+    let item = std::env::args().nth(3).expect("Please specify an item");
+    
+    println!("size of the filter {:?}", size);
+
+    let mut cuckoo_filter = CuckooFilter::<String>::new(size.parse::<usize>().unwrap());
+
+    if action == "add"{
+        cuckoo_filter.insert(&item);
+        match add(item) {
+            Ok(_) => println!("Item added"),
+            Err(why) => println!("An error occured: {}", why),
+        }
+
+    }
+
+    else if action == "delete"{
+        cuckoo_filter.remove(&item);
+        match delete(item) {
+            Ok(_) => println!("Item deleted"),
+            Err(why) => println!("An error occured: {}", why),
+        }
+
+    }
+       
+}
+
+
+fn add(item: String) -> redis::RedisResult<()> {
+    let client = redis::Client::open("redis://127.0.0.1/")?;

```

Comment:
**[Required, Performance]** We shouldn't be creating a new client connection every time a command is invoked. Have a persistent connection that is established on init for the module and reference it here instead of creating a new one.